src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/RazorBaselineIntegrationTestBase.cs

```diff
@@ -154,6 +154,33 @@ protected void AssertCSharpDocumentMatchesBaseline(RazorCodeDocument codeDocumen
             var actualMappings = SourceMappingsSerializer.Serialize(document, codeDocument.Source);
             actualMappings = actualMappings.Replace("\r", "").Replace("\n", "\r\n");
             Assert.Equal(baselineMappings, actualMappings);
+
+            AssertLinePragmas(codeDocument);
+        }
+
+        protected void AssertLinePragmas(RazorCodeDocument codeDocument)
```

Comment:
Did we find any bugs in old Razor?

Reply:
Not for the design time tests atleast. This could also mean we missed a source mapping somewhere because that is our source of truth. But that's unlikely.

Reply:
I did come across some inconsistencies with how we generate code for `@model` directives when investigating https://github.com/aspnet/AspNetCore/issues/8790. I need to dig more to see if we missed something.